### ISSUE TITLE
AsciiDoctor table [width] setting not implemented in vuetify output

### ISSUE THREAD

**Issue by tjquinno** [oracle/helidon-build-tools#31]
It looks like a table width setting using
```
[width=75%]
|===
...
```
is ignored in the vuetify output.

**Comment by romain-grecourt** [comment]
The current implementation of table in the vuetify output ignores a lot of options. The current way to set the width of a table is by using roles and add custom css classes.

E.g.
```
[role="flex, sm7"]
|=======
...
```

However implementing width should be relatively trivial I don't see a reason not to.

**Comment by tjquinno** [comment]
Yes, I just wanted to create an issue to track this. Nothing urgent.
